requirements.txt

```diff
@@ -1,9 +1,9 @@
-django>=2.2,<3.1
+django>=2.2,<3.2
```

Comment:
we need to update `.travis.yml` as well to test django 3.1, can you do that please?

Reply:
Done! :smile:

Reply:
I have undone this change. 
I just realized that openwisp-users is not 3.1 compatible yet, we will have to do that first!